src/main/java/com/khutircraftubackend/auth/AuthResponseMessages.java

```diff
@@ -0,0 +1,18 @@
+package com.khutircraftubackend.auth;
+
+public final class AuthResponseMessages {
```

Comment:
just for your information
i saw some times such class with static constants create as an interface
more about it with examples here: 
https://en.wikipedia.org/wiki/Constant_interface